internal/pkg/cli/env_deploy.go

```diff
@@ -134,8 +137,16 @@ func (o *deployEnvOpts) Execute() error {
 		RootUserARN:         caller.RootUserARN,
 		CustomResourcesURLs: urls,
 		Manifest:            mft,
+		ForceNewUpdate:      o.forceNewUpdate,
 		RawManifest:         rawMft,
 	}); err != nil {
+		var errEmptyChangeSet *awscfn.ErrChangeSetEmpty
+		if errors.As(err, &errEmptyChangeSet) {
+			log.Errorf(`It seems like your update does not introduce immediate resource changes. 
+This may because the resources being modified are not created yet until deemed necessary by the services in your environment.
+In this case, you can try %s. This will deploy the modified template, even if there is no immediate changes.
+`, color.HighlightCode("copilot env deploy --force"))
+		}
```

Comment:
```suggestion
			log.Errorf(`Your update does not introduce immediate resource changes. 
This may be because the resources are not created until they are deemed 
necessary by a service deployment.

In this case, you can run %s to push a modified template, even if there are no immediate changes.
`, color.HighlightCode("copilot env deploy --force"))
		}
```